src/available-command.ts

```diff
@@ -347,11 +347,11 @@ export function possibleFreeActions(engine: Engine, player: Player) {
     commands.push({
       name: Command.BurnPower,
       player,
-      data: _.range(1, Math.floor(engine.player(player).data.power.area2 / 2) + 1)
+      data: _.range(1, engine.player(player).data.burnablePower() + 1)
```

Comment:
Is it correct +1? If you have 1 you can burn 0, 2 => 1, 3 => 1

Reply:
Yes. _.range is [start, end). It will go from start to end-1.

Reply:
counterintuitive ;)